deepchem/models/losses.py

```diff
@@ -976,6 +976,40 @@ def loss(node_emb, inputs):
         return loss
 
 
+class GraphNodeMaskingLoss(Loss):
```

Comment:
Can you add docstring with usage example?

Reply:
done